accord-core/src/main/java/accord/topology/Topology.java

```diff
@@ -147,11 +179,6 @@ public boolean isSubset()
         return supersetIndexes.length < shards.length;
     }
 
-    public Topology withEpoch(long epoch)
```

Comment:
moved to `TopologyUtils` under `src/test` as the semantics are not clear what to do here... *should* this also update the `global` reference?  Since this was only used by tests and only for the `global` state anyways felt was easier to punt figuring out the semantic and just make it a test method